### PR TITLE
Add context documentation

### PR DESCRIPTION
**Pull request by imurchie**
## Proposed changes

Add documentation for context commands, and update the hybrid/web and related docs.

One small tweak to the template, to make types in parameters and responses be in `code` blocks, so formatting remains (so we get things like the `Array<String>` designation).

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: commands-yml/commands/context/get-context.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    For information on contexts, see Appium's [hybrid automation docs](/docs/en/writing-running-appium/web/hybrid.md).
+
+example_usage:
+  java:
+    |
+      String context = driver.getContext();
+  python:
+    |
+      context = driver.current_context
+      # of
```
Comment (dpgraham) [inline commands-yml/commands/context/get-context.yml]: #or

File: commands-yml/commands/context/get-contexts.yml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      contexts = @driver.available_contexts
+  php:
+    |
+      $contexts = $driver->contexts();
+  csharp:
+    |
+      // TODO C# sample
+
+client_docs:
+  java: "https://github.com/appium/java-client"
```
Comment (dpgraham) [inline commands-yml/commands/context/get-contexts.yml]: http://appium.github.io/java-client/io/appium/java_client/AppiumDriver.html#getContextHandles--

File: commands-yml/commands/context/get-context.yml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+      context = @driver.current_context
+  php:
+    |
+      $context = $driver->context();
+  csharp:
+    |
+      // TODO C# sample
+
+client_docs:
+  java: "https://github.com/appium/java-client"
```
Comment (dpgraham) [inline commands-yml/commands/context/get-context.yml]: http://appium.github.io/java-client/io/appium/java_client/AppiumDriver.html#getContext--

File: commands-yml/commands/context/set-context.yml [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+      $contexts = $driver->contexts();
+      $driver->context($contexts[1]);
+      // ...
+      $driver->context('NATIVE_APP');
+  csharp:
+    |
+      // TODO C# sample
+
+client_docs:
+  java: "https://github.com/appium/java-client"
```
Comment (dpgraham) [inline commands-yml/commands/context/set-context.yml]: http://appium.github.io/java-client/io/appium/java_client/AppiumDriver.html#context-java.lang.String-

**Review by dpgraham — COMMENTED** [review]
Looks good, just a few comments

**Comment by vmaxim** [comment]
And please fix wrong endpoints

File: commands-yml/commands/context/get-contexts.yml [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+  ruby: true
+  php: true
+  csharp: true
+  javascript_wd: true
+  javascript_wdio: true
+
+
+# Information about the HTTP endpoints
+endpoint:
+  url: /wd/hub/session/:session_id/element/value
```
Comment (vmaxim) [inline commands-yml/commands/context/get-contexts.yml]: invalid endpoint

File: commands-yml/commands/context/set-context.yml [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+  ruby: true
+  php: true
+  csharp: true
+  javascript_wd: true
+  javascript_wdio: true
+
+
+# Information about the HTTP endpoints
+endpoint:
+  url: /wd/hub/session/:session_id/element/value
```
Comment (vmaxim) [inline commands-yml/commands/context/set-context.yml]: incorrect endpoint

File: commands-yml/commands/context/get-context.yml (hunk 1 above)
Comment (imurchie) [reply commands-yml/commands/context/get-context.yml]: Damn. I fixed that once!

File: commands-yml/commands/context/get-context.yml (hunk 3 above)
Comment (imurchie) [reply commands-yml/commands/context/get-context.yml]: Hmm. That's definitely some pointless documentation.

**Comment by imurchie** [comment]
Updated.

**Review by SrinivasanTarget — APPROVED** [review]

**Pull request merged**